src/sctools/fastq.py

```diff
@@ -242,6 +242,6 @@ def extract_cell_barcode(self, record, cb):
         seq_tag, qual_tag = extract_barcode(record, cb)
         try:
             corrected_cb = self._error_mapping.get_corrected_barcode(seq_tag[1])
-            return seq_tag, qual_tag, ('CB', corrected_cb, 'Z')
         except KeyError:
-            return seq_tag, qual_tag
+            corrected_cb = seq_tag[1]  # couldn't find the right barcode!
+        return seq_tag, qual_tag, ('CB', corrected_cb, 'Z')
```

Comment:
Not a big deal, but no newline here at the end.

Reply:
Fixed.